modules/ossm-rn-known-issues.adoc

```diff
@@ -34,6 +34,15 @@ These are the known issues in {SMProductName}:
 
 * link:https://github.com/istio/istio/issues/14743[Istio-14743] Due to limitations in the version of Istio that this release of {SMProductName} is based on, there may be applications that are currently incompatible with {SMProductShortName}. See the linked community issue for details.
```

Comment:
This link to upstream bug can be removed. It's no longer an issue and had been closed for a while.